README.md

```diff
@@ -93,7 +113,6 @@ def my_commit_hook(event: FSEvent, ctx: HookContext) -> CommitCreation:
         message = f"❌ Remove file {ctx.resource}"
     else:
         message = f"✅ Add file {ctx.resource}"
-    
```

Comment:
Please revert, the space makes it more readable because of the different indentations (imo).